README.md

```diff
@@ -27,14 +30,32 @@ Find another group to work with. The other group should be students who neither
 (7 minutes) The Designer's first step in design thinking is empathizing with the person who the design is for.  The Designer should have a conversation with the Customer to get to know them and to ask about the Customer's needs and interests.  The goal is to try to identify the customer's need, *not a specific solution*.  The Designers who are going first should interview the customer and try to figure out the underlying need they'd like to satisfy.  
 
 ***Q1.2 Share some notes from your interview here and try to describe the "need" with a very brief description:***
+* Determine if there is too much of some chemicals in the blood
+* Check the user's blood pressure and heart rate
+* Identify the symptoms of the overdose
+  * Could lead to false alarms
+* Could use hardware that users already have like a fitbit
+* Phone app to track and store data to the cloud
+
 
 (6 minutes) Reverse roles. The Customers are now Designers and vice versa.  Repeat the interview process. (Designers, complete the question above in that space)
+Product: Fix bad handwriting with a mechanical solution
 
 ## Part 1.4: Proposing solutions (8 minutes)
 
 Return to your original locations.  In this step all groups will act like Designers.  Another vital aspect of design thinking is open-minded brainstorming about potential solutions that will fulfill the Customer's needs.   Review your notes about the Customer's needs.  Use cards from the Things, Services, Human Actions, and Feedback decks to *design two (2) or more different solutions* to the Customer's needs.
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
+1. A device that would attach to the user's arm to assist in writing
```

Comment:
This is definitely intriguing! I'm thinking about a writing device that could project characters onto paper as you write.